db/migrate/20180403171730_change_id_to_auto_increment_in_evaluations.rb

```diff
@@ -0,0 +1,20 @@
+class ChangeIdToAutoIncrementInEvaluations < ActiveRecord::Migration
```

Comment:
Missing this migration from structure.sql.
If you edit it manually, please add this line to the end of file:
`INSERT INTO schema_migrations (version) VALUES ('20180403171730');`